QUEST_LV_0100.tsv

```diff
@@ -3107,7 +3107,7 @@ QUEST_LV_0100_20150317_003102	I will go meet Laimondas
 QUEST_LV_0100_20150317_003103	Church Gate (1)
 QUEST_LV_0100_20150317_003104	Church Gate (2)
 QUEST_LV_0100_20150317_003105	I'll open the front door
-QUEST_LV_0100_20150317_003106	To entrance of Kateen Forest
+QUEST_LV_0100_20150317_003106	To Entrance of Kateen Forest
```

Comment:
I know! But but English grammar xD
We do say Go to the White House. not Go to White House. :P